cocos/gfx/base/pipeline-state.editor.ts

```diff
@@ -48,7 +48,16 @@ import {
 
 import { EffectAsset } from '../../asset/assets/effect-asset';
 import { RenderPassStage } from '../../rendering/define';
-import { CCString } from '../../core';
+import { CCString, ccenum } from '../../core';
+
+ccenum(PolygonMode);
```

Comment:
@star-e Are these objects external? 

We cannot modify the type of objects that are open to the public. If it is an external object, a new object needs to be declared here.